src/com/facebook/buck/ide/intellij/IjProjectTemplateDataPreparer.java

```diff
@@ -503,6 +513,41 @@ private void addAndroidResourcePaths(
     }
   }
 
+  private void addAndroidCompilerOutputPath(
+      Map<String, Object> androidProperties,
+      IjModule module,
+      Path moduleBasePath) {
+    // This is the fallback default in case we can't find a valid build target name
+    String targetName = "src_release";
+
+    if (module.getTargets() != null) {
+      for (BuildTarget target : module.getTargets()) {
+        BuildRule buildRule = buildRuleResolver.getRule(target);
+        if ((buildRule instanceof AndroidLibrary
```

Comment:
Can we just do this for any JvmLibrary. That way it will work for android,java,kotlin,groovy rules etc without any extra configuration